src/commands/moderation.rs

```diff
@@ -312,35 +317,43 @@ pub async fn mute(
         pending_unmute.abort();
     }
 
-    data.pending_unmutes.insert(
-        member.user.id.0,
-        queue_unmute_member(
-            &ctx.discord().http,
-            &data.database,
-            &member,
-            mute_role_id,
-            mute_duration.num_seconds() as u64,
-        ),
-    );
+    if !is_eternal {
+        data.pending_unmutes.insert(
+            member.user.id.0,
+            queue_unmute_member(
+                &ctx.discord().http,
+                &data.database,
+                &member,
+                mute_role_id,
+                mute_duration.num_seconds() as u64,
+            ),
+        );
+    }
 
     if result.is_none() {
         if let Err(e) = member.disconnect_from_voice(&ctx.discord().http).await {
             warn!("Could not disconnect member from voice channel: {}", e);
         }
     }
 
+    let duration = if is_eternal {
+        "Never.".to_string()
```

Comment:
Instead of this, simply do not append the duration field to the embed.
```suggestion
        "Indefinitely.".to_string()
```